### ISSUE TITLE
Research: Investigate why aXe 4.0 doesn't work with calcite-color

### ISSUE THREAD

**Issue by driskull** [Esri/calcite-components#831]
### Background 

Updating the aXe dependency to version 4.0.0 causes a failure with calcite-color. Some investigation needs to be done to see if it can be fixed or if it is an issue with the new version.

### Desired Outcome 

A fix for calcie color or an issue in aXe

**Comment by jcfranco** [comment]
Installed!

**Comment by driskull** [comment]
👍
